modules/solid_mechanics/src/auxkernels/MaterialTensorCalculator.C

```diff
@@ -13,9 +13,10 @@ InputParameters
 validParams<MaterialTensorCalculator>()
 {
   InputParameters params = emptyInputParameters();
-  MooseEnum quantities("VonMises=1 PlasticStrainMag Hydrostatic Direction Hoop Radial Axial "
-                       "MaxPrincipal MedPrincipal MinPrincipal FirstInvariant SecondInvariant "
-                       "ThirdInvariant TriAxiality VolumetricStrain");
+  MooseEnum quantities(
+      "VonMises=1 PlasticStrainMag EffectiveStrain Hydrostatic Direction Hoop Radial Axial "
```

Comment:
Since this will provide the correct way to compute `PlasticStrainMag`, maybe you should generate a warning if they're asking for it in this commit (although that will probably break a bunch of tests, so maybe you want to do that in the next round of changes).